src/lv_conf_internal.h

```diff
@@ -2173,6 +2173,24 @@
     #endif
 #endif
 
+/*API for Arduino LittleFs. */
```

Comment:
```suggestion
/*API for Arduino ESP LittleFS. */
```

Reply:
The comment is not corrected yet, but config name reflects that it's for Arduino ESP, so OK.